src/vnodeTransformers/stubComponentsTransformer.ts

```diff
@@ -117,8 +117,14 @@ export const createStub = ({
         // Also having function text as attribute is useless and annoying so
         // we replace it with "[Function]""
         const stubProps = normalizeStubProps(props)
-
-        return h(tag, stubProps, renderStubDefaultSlot ? slots : undefined)
+        // if renderStubDefaultSlot is true, we render the default slot
+        if (renderStubDefaultSlot) {
+          // we explicitly call the default slot with an empty object
+          // so scope slots destructuring works
+          const slot = slots.default ? slots.default({}) : slots
+          return h(tag, stubProps, slot)
+        }
```

Comment:
This could be simplified to:
```suggestion
        // if renderStubDefaultSlot is true, we render the default slot
        if (renderStubDefaultSlot && slots.default) {
          // we explicitly call the default slot with an empty object
          // so scope slots destructuring works
          return h(tag, stubProps, slots.default({}))
        }
```

If `slots.default` is not set, there is no need to pass `slots`

Reply:
I think that, weirdly enough, a test was breaking if I did not. But I agree so I'll check again when I'm on my laptop 👍